### DISCUSSION TITLE
How to export NVChad as nvim when files are being loaded form .config/nvim-chad?

### DISCUSSION THREAD

**Discussion by moshpirit** [NvChad/NvChad#2800] [Q&A] [answered]
I am trying different distros of nvim because I like to try different things and see which one I like the most; so I want to be able to change the default nvim distro just from one place: `.bashrc`, and use it as `$EDITOR`, so I thought about exporting it from there like this:

```bash
# choose between nvim-chad, nvim-lazy and nvim-kickstart
nvim_distro='nvim-chad' # I also have nvim-lazy and nvim-kickstart
alias nvim="NVIM_APPNAME=\"$nvim_distro\" nvim"
export EDITOR="nvim -u $HOME/.config/$nvim_distro/init.lua"
export nvim_distro
```

everything works fine but if I use `$EDITOR`, I get neovim opened but I also I get this error message: 
```
An error has been found when processinhg /home/alberto/.config/nvim-chad/init.lua:
E5113: Error while calling lua chunk: cannot open /home/alberto/.local/share/nvim/nvc
had/base46/defaults: No such file or directory
stack traceback:
        [C]: in function 'dofile'
        /home/alberto/.config/nvim-chad/init.lua:21: in main chunk
```

that belongs to:

```lua
dofile(vim.g.base46_cache .. "defaults")
vim.opt.rtp:prepend(lazypath)
require "plugins"
```
I've searched the file all over my computer but I couldn't find it. I installed NVChad from AUR, because I don't know how to keep programs updated when I git clone

**Comment by siduck** [comment]
never install nvchad from aur.

just clone it in `~/.config/nvchad`

```bash
alias nv='NVIM_APPNAME="nvchad" nvim`
```

**Reply by moshpirit** [reply]
Hi, @siduck, thanks for replying!

>never install nvchad from aur.
>just clone it in `~/.config/nvchad`

Why not AUR? How can I keep chad updated then?

**Reply by siduck** [reply, marked as answer]
@moshpirit from nvchad v2.5 you can just use lazy sync command to update nvchad. 

Because the starter repo teaches you how to use the main repo of nvchad itself as a plugin and import its mappings, autocmds etc

**Reply by moshpirit** [reply]
Thanks for replying! I hear you and did what you said (cloning the repo), although I still can't export it as the default configuration for  `$EDITOR`.

**Reply by siduck** [reply]
@moshpirit  make editor value same as the alias

**Reply by moshpirit** [reply]
I couldn't make it work by using this:
```
alias nv='NVIM_APPNAME="nvim-chad" nvim'
export EDITOR=nv
```
or `export EDITOR='NVIM_APPNAME="nvim-chad" nvim'`, but this did!:

``` 
export NVIM_APPNAME="nvim-chad"
export EDITOR=nvim
```

Thank you for your time and for maintaining NvChad!! It's fantastic!
